test/e2e-v2/cases/promql/promql-cases.yaml

```diff
@@ -58,6 +58,9 @@ cases:
     expected: expected/service-metric-sort-vector.yml
   - query: curl -X GET http://${oap_host}:${oap_9090}/api/v1/query -d 'query=service_cpm{layer="GENERAL",top_n="10", order="DES"}[30m]'
     expected: expected/service-metric-sort-matrix.yml
+  ## multiple metrics operation query
+  - query: curl -X GET http://${oap_host}:${oap_9090}/api/v1/query -d 'query=service_sla{service="e2e-service-consumer", layer="GENERAL"} + service_cpm{service="e2e-service-consumer", layer="GENERAL"}'
+    expected: expected/service-metric-vector.yml
```

Comment:
I think we need to add a test case with labeled metrics.

Reply:
@mrproliu Could you add one for percentile with a specific label? Such as `p99`?

Reply:
> get specific label from PromQL

What do you mean get specific label?

Reply:
I thought that you want to make a judgment by only reading a specific label from a certain metric? For example, if there are {label1: "1", label2: "2"}, you would like to compare only the metrics with "label1" and ignore "label2".

Reply:
`service_percentile{service="e2e-service-consumer", layer="GENERAL", labels="0,1,2", relabels="P50,P75,P90"}` is select `0,1,2` and relabel them.

Reply:
I mean you could do `service_percentile{service="e2e-service-consumer", layer="GENERAL", labels="4"} - service_percentile{service="e2e-service-consumer", layer="GENERAL", labels="3"}` which mean the gap between p99 and p95 to see whether the long tail of the metric exists or not?

Reply:
I understand now. I was trying to determine if they have the same labels but different name metrics, so I misunderstood your intention.

Reply:
Sorry my fault, it should be 

> service_percentile{service="e2e-service-consumer", layer="GENERAL", labels="4", relabels"p_value"} - service_percentile{service="e2e-service-consumer", layer="GENERAL", labels="3", relabels"p_value"}

The label should be aligned manually.

Reply:
`service_percentile{service="e2e-service-consumer", layer="GENERAL", labels="4", relabels="P99_P55"} - service_percentile{service="e2e-service-consumer", layer="GENERAL", labels="3", relabels="P99_P55"}`